documentation/config.md

```diff
@@ -37,9 +44,13 @@ action.url = http://127.0.0.1:9090/actions/do
 
 # Time in milliseconds (long) to "expire" a "dangling" rule
 rule.max_age= 60000
+
+# millisecond resolutuion of the internal timer thread
+internal_timer_msec_resolution = 30
 ```
 
 The `action.url` field is the URL used to make a POST in the case of a rule being fired by an incoming event. The content POSTed is the JSON representation of the derived  ("complex") event.
 
 The `rule.max_age` field is the value in milliseconds used as a threshold for the age of a rule to deleted safely in case of a PUT request over the rules resource does not contain the rule. All the rules older than rule.max_age will be deleted if they are not in the new set. If a rule is younger than `rule.max_age` it will be kept despite not being in the set sent by the PUT request.
 
+The `internal_timer_msec_resolution` field is the value in milliseconds used for resolution of the internal timer thread
```

Comment:
Maybe we should explain what can be happen if we use a very small value for this (e.g. does it have some impact in performance)? Does Esper docuementation tells something about this?

Reply:
updated in https://github.com/telefonicaid/perseo-core/pull/195/commits/6851c32b4abb8b331b5cc476b48962777350b9f6